cartography/intel/aws/ses.py

```
@@ -0,0 +1,278 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+import boto3
+import neo4j
+
+from cartography.client.core.tx import load
+from cartography.graph.job import GraphJob
+from cartography.models.aws.ses.identity import SESIdentitySchema
+from cartography.models.aws.ses.configuration_set import SESConfigurationSetSchema
+from cartography.stats import get_stats_client
+from cartography.util import aws_handle_regions
+from cartography.util import merge_module_sync_metadata
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+stat_handler = get_stats_client(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_ses_identities(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    """
+    Get all SES identities (email addresses and domains) for a region.
+    """
+    client = boto3_session.client("ses", region_name=region)
+    try:
+        response = client.list_identities()
+        return response.get("Identities", [])
+    except Exception as e:
```

For error handling, we prefer to fail loudly. It’s better not to catch the exception here and let it propagate.